wgpu/src/backend/mod.rs

```diff
@@ -1,12 +1,32 @@
-#[cfg(all(target_arch = "wasm32", not(feature = "webgl")))]
+#[cfg(all(
+    target_arch = "wasm32",
+    not(target_os = "emscripten"),
```

Comment:
similarly, this configuration doesn't need to change. If emscripten path requires "webgl" feature, it mostly follows the same steps.